osmosis-1/osmosis-frontier.assetlist.json

```diff
@@ -1670,6 +1670,35 @@
         "svg": "https://raw.githubusercontent.com/osmosis-labs/assetlists/main/images/hash.svg"
       },
       "coingecko_id": "provenance-blockchain"
+    },
+    {
+      "description": "GLX is the staking token of the Galaxy Chain",
+      "denom_units": [
+        {
+          "denom": "ibc/F49DE040EBA5AB2FAD5F660C2A1DDF98A68470FAE82229818BE775EBF3EE79F2",
+          "exponent": 0,
+          "aliases": [
+            "uglx"
+          ]
+        },
+        {
+          "denom": "glx",
+          "exponent": 6
+        }
+      ],
+      "base": "ibc/F49DE040EBA5AB2FAD5F660C2A1DDF98A68470FAE82229818BE775EBF3EE79F2",
+      "name": "Glx",
+      "display": "glx",
+      "symbol": "GLX",
+      "ibc": {
+        "source_channel": "channel-0",
+        "dst_channel": "channel-236",
+        "source_denom": "uglx"
+      },
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/osmosis-labs/assetlists/main/images/glx.png",
```

Comment:
Please point this to Chain registry
https://github.com/cosmos/chain-registry/blob/master/galaxy/images/glx.png
and
https://github.com/cosmos/chain-registry/blob/master/galaxy/images/glx.svg

Reply:
I have make change to chain registry